### PR TITLE
Normalize ES6 const declarations to preserve inline type annotations.

### LINKED ISSUE #1314: Unlike var statements, const statements do not support multiple types
Code -
```
// ==ClosureCompiler==
// @output_file_name default.js
// @compilation_level ADVANCED_OPTIMIZATIONS
// @language ECMASCRIPT6_STRICT
// ==/ClosureCompiler==
"use strict";
const
  /** @const {string} */ foo = "1",
  /** @const {number} */ bar = 1;

const
  /** @type {string} */ baz = "1",
  /** @type {number} */ taz = 1;
```

Warnings -
```
JSC_MULTIPLE_VAR_DEF: declaration of multiple variables with shared type information at line 2 character 0
const
^
JSC_MULTIPLE_VAR_DEF: declaration of multiple variables with shared type information at line 6 character 0
const
^
```

Note - changing them to `var` avoids the warnings, of course.

### PR DESCRIPTION
**Pull request by Dominator008**
Fixes #1314

### CODE REVIEW

File: src/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclaration.java [hunk 1]
```diff
@@ -132,17 +132,26 @@ private void varify() {
...
       for (Node n : letConsts) {
         if (n.isConst()) {
-          JSDocInfo existingInfo = n.getJSDocInfo();
-          if (existingInfo == null) {
-            existingInfo = n.getFirstChild().getJSDocInfo();
-            n.getFirstChild().setJSDocInfo(null);
+          // Normalize declarations like "const x = 1, y = 2;" so that inline
+          // type annotations are preserved.
+          for (Node child : n.children()) {
+            Node declaration = new Node(Token.VAR, child.detachFromParent());
```
Comment (MatrixFrog) [inline src/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclaration.java]: nit: IR.var() just because it's shorter.

**Comment by MatrixFrog** [comment]
otherwise lgtm

File: src/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclaration.java (hunk 1 above)
Comment (Dominator008) [inline src/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclaration.java]: I thought IR.var() takes a lhs, so I will need to extract that from "child"?

Comment (MatrixFrog) [inline src/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclaration.java]: `Node declaration = IR.var(child.detachFromParent());` is equivalent no?

Comment (Dominator008) [inline src/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclaration.java]: Done. I got confused..

**Comment by Dominator008** [comment]
@MatrixFrog This should be ready now.

**Pull request merged**